ludwig/schema/hyperopt/__init__.py

```diff
@@ -0,0 +1,83 @@
+from abc import ABC
+from typing import Dict
+
+from marshmallow_dataclass import dataclass
+
+from ludwig.constants import LOSS, TEST, TRAIN, VALIDATION
+from ludwig.schema import utils as schema_utils
+from ludwig.schema.features.utils import output_config_registry
+from ludwig.schema.hyperopt.executor import ExecutorConfig, ExecutorDataclassField
+from ludwig.schema.hyperopt.search_algorithm import BaseSearchAlgorithmConfig, SearchAlgorithmDataclassField
+
+
+@dataclass
+class HyperoptConfig(schema_utils.BaseMarshmallowConfig, ABC):
+    """Basic hyperopt settings."""
+
+    def get_hyperopt_metric_options():
+        all_metrics = []
+        for oftype in output_config_registry:
+            ofcls = output_config_registry[oftype]
+            all_metrics += ofcls.get_output_metric_functions().keys()
+        return all_metrics
+
+    output_feature: str = "combined"  # TODO: make more restrictive
```

Comment:
Does this mean that it can be any string at the moment? That's totally ok for now, just want to make sure!